jelly/src/test/java/org/kohsuke/stapler/jelly/BindTagTest.java

```diff
@@ -23,19 +24,19 @@ protected void setUp() throws Exception {
 
 
     public void test1() throws Exception {
-        WebClient wc = new WebClient();
+        WebClient wc = createWebClient();
         HtmlPage page = wc.getPage(new URL(url, "/"));
         String content = page.getWebResponse().getContentAsString();
         System.out.println(content);
-        //Check that prototype is included in the page
-        assertTrue(content.contains("/am/org/kohsuke/stapler/framework/prototype/prototype.js"));
-        page.executeJavaScript("v.foo('hello world');");
+        page.executeJavaScript("v.foo('hello world', 2);");
         wc.getJavaScriptEngine().processPostponedActions();
         wc.waitForBackgroundJavaScript(10000);
         assertEquals("hello world",value);
+        assertEquals(2, number);
     }
 
-    public void jsFoo(String arg) {
+    public void jsFoo(String arg, int arg2) {
```

Comment:
This test was copied to a jQuery version, which was later enhanced by supporting an extra argument (probably to test some additional use case). I am now deleting the jQuery version of this test, but I thought I'd keep this test enhancement and backport it to the original test, since if someone thought it worthwhile to enhance the other test, that same logic probably applies here as well.